src/components/sections/ProductGallery/ProductGallery.tsx

```diff
@@ -1,23 +1,22 @@
+import './product-gallery.scss'
```

Comment:
Are there some gains on importing the style file first? If so we can replicate it in the others components.

Reply:
ops, not really, This is just my code editor sorting imports.

Reply:
I'm reverting this change. Also, I'm resolving this comment